docs/rfc-identity.md

```diff
@@ -0,0 +1,186 @@
+# RFC Peer-Star-Identity
+
+This is a proposal for an identity management system for Peer-Star, named Peer-Star-Identity. It's built upon standards to provide a good foundation for Peer-Star applications to authenticate and identify users.
+
+*Authors*: André Cruz, João Santos, Pedro Teixeira
+
+1. [Standards & foundations](#standards--foundations)
+   1. [Decentralized identifiers](#decentralized-identifiers-dids)
+   1. [Self-signed Verifiable Claims](#self-signed-verifiable-claims)
+   1. [DID Delegates & devices](#did-delegates--devices)
+   1. [Using DID-Auth to prove control of the DID](#using-did-auth-to-prove-control-of-the-did)
+1. [Proposal: The IdentityManager](#proposal-the-identitymanager)
+   1. [What is it?](#what-is-it)
+   1. [Managing identities](#managing-identities)
+   1. [Managing Verifiable Claims](#managing-verifiable-claims)
+   1. [Authenticating on applications](#authentication-on-applications)
+   1. [Signing artifacts on applications](#signing-artifacts-on-applications)
+   1. [Managing application sessions](#managing-application-sessions)
+   1. [Revoking a device](#revoking-a-device)
+1. [Diagrams & mockups](#diagrams--mockups)
+1. [Glossary](#glossary)
+
+## Standards & foundations
+
+### Decentralized identifiers (DIDs)
```

Comment:
@jonnycrunch, @b5, @ianopolous would love your review on this RFC, I know that you have been thinking about Identity on the Decentralized Web a ton :)

Reply:
@lanzafame just read up https://github.com/ipfs/notes/issues/292, you should review this one too! :)